server/src/test/java/com/google/coffeehouse/storagehandler/StorageHandlerTestHelper.java

```diff
@@ -111,190 +139,135 @@ public static void insertBook(String book_id) {
-        .set("bookId")
-        .to(book_id)
+        .to("isbn")
         .set("title")
-        .to(title)
+        .to("title")
         .build());
     dbClient.write(mutations);
   }
 
-  public void insertBookWithEmptyAuthor(String book_id, String isbn, String title) {
+  public static void insertBookWithNullIsbn(String book_id) {
     List<Mutation> mutations = new ArrayList<>();
     mutations.add(
       Mutation.newInsertOrUpdateBuilder("Books")
         .set("bookId")
         .to(book_id)
         .set("author")
-        .to("")
-        .set("isbn")
-        .to(isbn)
+        .to("author")
         .set("title")
-        .to(title)
+        .to("title")
         .build());
     dbClient.write(mutations);
   }
 
-  public void insertBookWithEmptyIsbn(String book_id, String author, String title) {
+  public static void insertBookWithNullAuthorAndNullIsbn(String book_id) {
     List<Mutation> mutations = new ArrayList<>();
     mutations.add(
       Mutation.newInsertOrUpdateBuilder("Books")
         .set("bookId")
         .to(book_id)
-        .set("author")
-        .to(author)
-        .set("isbn")
-        .to("")
         .set("title")
-        .to(title)
+        .to("title")
         .build());
     dbClient.write(mutations);
   }
 
-  public void insertBookWithEmptyAuthorAndEmptyIsbn(String book_id, String title) {
-    List<Mutation> mutations = new ArrayList<>();
-    mutations.add(
-      Mutation.newInsertOrUpdateBuilder("Books")
-        .set("bookId")
-        .to(book_id)
-        .set("author")
-        .to("")
-        .set("isbn")
-        .to("")
-        .set("title")
-        .to(title)
-        .build());
-    dbClient.write(mutations);
+  public static Person createTestPersonObject(String person_id, Boolean pronouns_exist) {
+    Person.Builder personBuilder = Person.newBuilder()
+                                         .setEmail("person@test.com")
+                                         .setNickname("person")
+                                         .setUserId(person_id);
+    if (pronouns_exist) {
+      personBuilder.setPronouns("she/he/they");
+    }
+    return personBuilder.build();
   }
 
-  public void insertBookWithEmptyAuthorAndNullIsbn(String book_id, String title) {
-    List<Mutation> mutations = new ArrayList<>();
-    mutations.add(
-      Mutation.newInsertOrUpdateBuilder("Books")
-        .set("bookId")
-        .to(book_id)
-        .set("author")
-        .to("")
-        .set("title")
-        .to(title)
-        .build());
-    dbClient.write(mutations);
+  public static Book createTestBookObject(String book_id, Boolean isbn_exists,
+                                            Boolean author_exists) {
+    Book.Builder bookBuilder = Book.newBuilder()
+                                   .setTitle("title")
+                                   .setBookId(book_id);
+    if (isbn_exists) { bookBuilder.setIsbn("isbn"); }
+    if (author_exists) { bookBuilder.setAuthor("author"); }
```

Comment:
Technically Google style guide only allows concise brackets when they're empty (https://google.github.io/styleguide/javaguide.html#s4.1.2-blocks-k-r-style). This should be reformatted to not use concise brackets.

Reply:
Got it, thanks for pointing this out. I was trying to save some space, but will change this to adhere to the style guide.